components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -89,5 +271,386 @@ module.exports = {
       };
       return await axios(config);
     },
+    /**
+     * Send a text message
+     *
+     * @param {String} chatId - Unique identifier for the target chat or
+     * username of the target channel (in the format `@channelusername`)
+     * @param {String} text - Text of the message to be sent, 1-4096 characters
+     * after entities parsing
+     * @param {Object} [opts] - An object containing additional configuration
+     * options for this method, as defined in [the API
+     * docs](https://core.telegram.org/bots/api#sendmessage)
+     * @returns The sent Message
+     */
+    async sendMessage(chatId, text, opts) {
+      return await this.sdk().sendMessage(chatId, text, opts);
```

Comment:
Seems like it's unecessary to have these `await` here. Only places I would call `await` is because either I would put it inside a `try/catch` or because there is a need to resolve a value before returning it
```suggestion
      return this.sdk().sendMessage(chatId, text, opts);
```